APIs/schemas/token_schema.json

```diff
@@ -34,34 +34,30 @@
     "scope": {
       "description": "A string containing a space-separated list of scopes associated with the token",
       "type": "string"
-    },
-    "x-nmos-api": {
-      "description": "An object containing the access permissions of the user for the listed NMOS APIs",
+    }
+  },
+  "patternProperties": {
+    "^x-nmos-[a-z]+$": {
```

Comment:
I think this is fine, since we've practised consistent lack of punctuation in API and resource paths, e.g. `netctrl` and `channelmapping` rather than anything with '-', etc.